### ISSUE TITLE
consider adding a chart with tooltip in examples

### ISSUE THREAD

**Issue by neevany** [palantir/plottable#3214]
i thought tooltips are not supported till i saw the tutorial

**Comment by danielbaak** [comment]
@neevany I'm not sure about current tooltip support. 
The tooltip tutorial (http://plottablejs.org/tutorials/tooltips/) is using plottable v2.7.0 and I couldn't replicate it with plottable v3.1.1.
The issue seems to stem from the fact that plottable switched from `svg` to `html` elements in version 3.

**Comment by hellochar** [comment]
Yeah sorry we need to upgrade the docs/website for 3.0. @danielbaak check out this fiddle for an updated tooltip example: http://jsfiddle.net/hellochar/3rf4vrnq/1/

The changes you have to make:

1. make example a div
2. use CSS to give it a width/height
3. use .attrs() instead of .attr() in d3

**Comment by danielbaak** [comment]
@hellochar thanks for the quick response. It worked like a charm.
